src/components/StepSuggest.vue

```diff
@@ -34,6 +34,11 @@
     beforeCreate() {
       this.$store.dispatch('suggest');
     },
+    mounted() {
+      if (this.suggestions.length === 0) {
+        this.$store.dispatch('nextStep');
+      }
+    },
```

Comment:
That's nice however most of the time Suggest Step is skipped before the request has loaded with suggested symptoms (usually when I pick some risk factors and click next right away in new interview)